spec/features/scheduling/scheduled_publishing_spec.rb

```diff
@@ -46,12 +47,15 @@ def then_i_see_the_edition_is_scheduled
     expect(page).to have_content(I18n.t!("schedule.scheduled.title"))
     visit document_path(@edition.document)
 
+    expect(page).to have_content(I18n.t!("user_facing_states.scheduled.name"))
+    expect(page).to have_content("Scheduled to publish at 9:00am on 22 June 2019")
+  end
+
+  def and_there_is_a_scheduled_history_event
+    click_on "Document history"
     within(".app-timeline-entry") do
```

Comment:
I assume we don't need this within now this is on a new page?

Reply:
Yep, it is now removed!